tests/dummy/app/templates/index.hbs

```diff
@@ -456,3 +456,84 @@
     <p>Warning: Nested lists doesn't work well with horizontal lists.</p>
   </div>
 </div>
+
+<div class="list-groups">
+  <div class="list-group-wrapper">
+    <h2>Occlusion Rendering</h2>
+
+    <p>
```

Comment:
Why are we having bare strings for content? Won't this require i18n?
Even if i18n is not required, pls have this texts flows from a constant.

Reply:
This is a 3d party library and customizing it to solve our use case, we cannot introduce i18n here. Anything within the 'tests' folder is only for develpoment phase & unit test cases for the addon. This will not have impact on the consuming application.
Of course, we can pass it as a constant from the component. But in my opinion it is not recommended here because this is an Open Source addon (where many contributors would be there). Hence, I have implemented in a way that matches the pattern with rest of the examples in this file.